Documentation/configuring-reporting-operator.md

```diff
@@ -147,6 +119,46 @@ curl "http://35.227.172.86:8080/api/v1/reports/get?name=cluster-memory-capacity-
 
 ### Openshift Authentication
 
+By default, the reporting API is secured with TLS and authentication. This is done by configuring the reporting-operator to deploy a pod containing both the reporting-operator's container, and a sidecar container running [Openshift auth-proxy](https://github.com/openshift/oauth-proxy).
+
+If you want to manually configure authentication in Openshift, disable it entirely, or want more information on the reporting-operator options, see the [manual setup](#manual-setup) section.
+
+In order to access the reporting API, the metering operator exposes a route. Once that route has been installed, you can run the command below to get the route's hostname.
+
+```
+METERING_ROUTE_NAME=$(oc -n $METERING_NAMESPACE get routes metering -o json | jq '.status.ingress[].host')
```

Comment:
Maybe `METERING_ROUTE_HOST` or `METERING_ROUTE_HOSTNAME`?

Reply:
Yeah, that was a poor choice for that environment variable name - will update to `METERING_ROUTE_HOSTNAME`.